test/decl/class/actor/conformance.swift

```diff
@@ -2,6 +2,8 @@
 
 // REQUIRES: concurrency
 
```

Comment:
```suggestion
// REQUIRES: distributed
```

Reply:
I think you need to add that

Reply:
@ktoso @LucianoPAlmeida I've moved the test to `decl/protocol/special/DistributedActor.swift` where it already had a test with distributed actor implementing a protocol and all the required imports.

Reply:
Lookin good, thank you